response_operations_ui/templates/collection_exercise/ce-sample-ci-section.html

```diff
@@ -1,66 +1,88 @@
+                        {
+                            "value": "Uploaded " + '<em>' + sample.ingestDateTime[:-8] + '</em>' + sample.ingestDateTime[-8:],
+                            "thClasses": "ons-u-fs-r",
+                            "numeric": "true"
+                        },
+                    ],
+                    
+                    "trs": [
+                        {
+                            "tds": [
+                                {
+                                    "value": "Total businesses",
+                                    "tdClasses": "ons-u-ta"
+                                },
+                                {
+                                    "value": sample.totalSampleUnits,
+                                    "name": "total-instruments",
+                                    "tdClasses": "ons-u-ta-right"
+                                },
+                                {
+                                    "value": ' ',
+                                    "name": "upload-sample-time",
+                                }
+                            ]
+                        },
+                        {
+                            "tds": [
+                                {
+                                    "value": "Collection instruments",
+                                    "tdClasses": "ons-u-ta"
+                                },
+                                {
+                                    "value": sample.expectedCollectionInstruments,
+                                    "name": "total-instruments",
+                                    "tdClasses": "ons-u-ta-right"
+                                },
+                                {
+                                    "value": ' ',
+                                    "name": "upload-sample-time",
+                                }
+                            ]
+                        }
+                    ]
+                })
+            }}
+        {% elif 'INIT' == sample.state %}
+            <div class="ons-u-mb-xl">
+                <div class="ons-u-fs-r">
+                    Loading&hellip; <span class="ons-u-ml-xs"><a href="/surveys/{{survey.shortName}}/{{ce.exerciseRef}}/view-sample-ci?show_msg=true">Refresh to see progress</a></span>
+                </div>
+            </div>
+        {% elif 'FAILED' == sample.state %}
+            <div class="ons-u-mb-xl">
+                {% call
+                    onsPanel({
+                        "type": "error",
+                        "classes": "ons-u-mb-l"
+                    })
+                %}
+                    {{ sample.notes }}
+                {% endcall %}
+                {% if not locked and surveyEditPermission %}
+                    <a class="ons-u-ml-xs" id="remove_sample_btn_2" href="{{ url_for('collection_exercise_bp.get_confirm_remove_sample', short_name=survey.shortName, period=ce.exerciseRef) }}">Remove</a>
```

Comment:
I think we need to be more explicit then just remove the sample page we say "Replace sample file" (although that is wrong anyway as it doesn't replace anything just takes you to a delete confirmation)

Reply:
I'll just remove this on both pages as it doesn't make sense to have it since the first page button won't take you to the remove sample page and the second page already has a button that does that